### PR TITLE
April eco docs2

### PR DESCRIPTION
**Pull request by BobStolzberg**
3x new KB articles: Flume, Hortonworks Hadoop, Call Me

### CODE REVIEW

**Comment by BobStolzberg** [comment]
Added some bonus material:  Bitnami GitLab Blueprint docs

File: Ecosystem Partners/getting-started-with-apex-datacom-call-me-essentials-edition-blueprint.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+- Requires the customer to enter a Public DNS in the Blueprint
+
+### Postrequisite
+- Use after the Free trial period will require CenturyLink SIP long distance (IPLD) account.  That account can  be ordered ahead or inline by coordinating with Apex Datacom. 
+
+### Deploy Apex Datacom Call Me Essentials Edition Blueprint
+Follow these step by step instructions to deploy a single server instance of Call Me Essentials Edition.
+
+Follow these step by step instructions to deploy a single server instance of Call Me Essentials Edition.
+1.	Locate the Blueprint in the Blueprint Library.
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-apex-datacom-call-me-essentials-edition-blueprint.md]: Needs a line break between 55 and 56 to correctly start the ordered list

File: Ecosystem Partners/getting-started-with-apex-datacom-call-me-essentials-edition-blueprint.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+### Postrequisite
+- Use after the Free trial period will require CenturyLink SIP long distance (IPLD) account.  That account can  be ordered ahead or inline by coordinating with Apex Datacom. 
+
+### Deploy Apex Datacom Call Me Essentials Edition Blueprint
+Follow these step by step instructions to deploy a single server instance of Call Me Essentials Edition.
+
+Follow these step by step instructions to deploy a single server instance of Call Me Essentials Edition.
+1.	Locate the Blueprint in the Blueprint Library. 
+- Starting from the CenturyLink Control Panel, navigate to the Blueprints Library.
+- [Apex Datacom Image](../images/ecosystem-apex-datacom-callme-a1.png)
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-apex-datacom-call-me-essentials-edition-blueprint.md]: add an `!` to mark it as an img. Right now it's just a link.

File: Ecosystem Partners/getting-started-with-apex-datacom-call-me-essentials-edition-blueprint.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+- Search for “Call Me” in the keyword search on the right side of the Blueprints Library page as shown in the image below.
+- [Apex Datacom Image](../images/ecosystem-apex-datacom-callme-a2.png)
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-apex-datacom-call-me-essentials-edition-blueprint.md]: needs `!`

File: Ecosystem Partners/getting-started-with-hortonworks-hadoop-blueprint.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+1.	Once verified, click on the ‘deploy blueprint’ button. You will see the deployment details along with an email stating the Blueprint is queued for execution.
+2.	This will kick off the blueprint deploy process and load a page to allow you to track the progress of the deployment.
+6.	Monitor the Activity Queue.
+* Monitor the Deployment Queue to view the progress of the blueprint.
+* You can access the queue at any time by clicking the Queue link under the Blueprints menu on the main navigation drop-down.
+* Once the blueprint completes successfully, you will receive an email stating that the blueprint build is complete. Please do not use the Hortonworks Server until you have received this email notification.
+
+### Accessing your Hortonworks Hadoop Server
+After your Blueprint deploys successfully, please follow these instructions to access your server:
+1.	Check your email to obtain Server Name and IP Address Login information
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-hortonworks-hadoop-blueprint.md]: needs line break to start the list

Comment (BobStolzberg) [inline Ecosystem Partners/getting-started-with-hortonworks-hadoop-blueprint.md]: Done - Thank you, Nathan!

File: Ecosystem Partners/getting-started-with-bitnami-gitlab-blueprint.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+### Pricing
+The costs associated with this Blueprint deployment are for the CenturyLink Cloud infrastructure only.  There are no Bitnami license costs or additional fees bundled in.
+
+### About Bitnami
+CenturyLink Cloud works with [Bitnami](http://www.bitnami.com) to provide open source software integrations to its customers.  Bitnami is a library of popular server applications and development environments that can be installed with one click, either in your laptop, in a virtual machine or hosted in the cloud. Bitnami takes care of compiling and configuring the applications and all of their dependencies (third-party libraries, language runtimes, databases) so they work out-of-the-box. The resulting packaged software (a 'stack') is then made available as native installers, virtual machines and cloud images. These Bitnami application packages provide a consistent, secure and optimized end-user experience when deploying any app, on any platform.
+
+### Frequently Asked Questions
+
+- #### Who should I contact for support?
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-bitnami-gitlab-blueprint.md]: remove `- `

**Pull request merged**